docs/src/examples/components/Menu/commonSteps.ts

```diff
@@ -0,0 +1,30 @@
+import { Menu } from '@stardust-ui/react'
+
+interface StepsOptions {
+  vertical?: boolean
+}
+
+const menuSelector = `.${Menu.className}`
+const selectors = {
+  menu: menuSelector,
+  item: (itemIndex: number) => `${menuSelector} li:nth-child(${itemIndex}) a`,
+}
+
+const commonSteps = [
```

Comment:
It there any reason to keep this as a separate variable?

Reply:
to illustrate it's not changing and we don't recreate this part of the array every time we call `getSteps`

Reply:
For me: it's not the right place for micro optimizations, but it will be much easier to read all steps in the single place. _NIT_

Reply:
fair enough, let's put them in one place